platform/services/account/chart/templates/autoscaler.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: {{ include "account-service.fullname" . }}
-  {{- if .Values.global.running_on_vm }}
+  {{- if or .Values.global.running_on_vm (eq .Values.global.installation_profile "lite") }}
```

Comment:
I think the autoscalers should be disabled for all services if using lite installation profile. The values configured there are not compatible with the lite resources and may result in scaling up pods even on system idle. Autoscaling was also mainly added for SaaS/multi-user setups, which doesn't really match with the light deployment.

Reply:
Thanks Jacek for the PR, can you apply Albert’s suggestion? I am also wondering if we still need the running_on_vm flag? Can we use Geti Lite for VMs instead?